components/voice_registry_system.md

```diff
@@ -246,7 +246,31 @@ By following the SRP, we achieve the following: <br>
 #### 1.0.6.1 Do we need central location for common words?
 //todo: #85
 #### 1.0.6.2 Who is the decision maker whether user's utterance is explicit or implicit invocation?
-//todo: #86
+The voice assistant system will decide if a user's utterance is explicit or implicit. For the initial release, VRS will only handle explicit invocations and the voice assistant system will disambiguate implicit invocations.  
+
+**Explicit Invocation Definition:** An invocation of a voice application where the name of the application is part of the invocation utterance. For example:
+- ```Ask Target, to add milk to my shopping list```  
+- ```Open Target```  
+- ```Talk to Target```  
+- ```Ask Target, what’s on sale```  
+- ```Add milk to my Target shopping list```  
```

Comment:
Per @jcstine 's comment, add some examples around John's bakery and an explanation on disambiguation in an explicit request.

Reply:
Updated to include a John's Bakery example:

There will be instances where there is more than one voice application with the same name. For example:
- ```Ask John's Bakery, when do you open```
  
There may be more than one voice application named *John's Bakery*. Indeed, there may be many different *John's Bakery* voice applications. If a user's explicit invocation results in multiple VRS matches, VRS will return all of the matching records. Included in the records will be the address and other metadata about the voice applications. The voice assistant will then use this information and other user information to disambiguate. Possible ways to disambiguate the above example include:
- Using the address to find the nearest *John's Bakery*
- Using the last launched *John's Bakery* application
- Asking the user which *John's Bakery* to use (and likely store this in user preferences for future use)
- Some other AI based algorithm to find the best option for the user